test/frontend/ProfileAdminView.test.js

```diff
@@ -0,0 +1,108 @@
+/* global describe it */
+import 'jsdom-global/register'
+import React from 'react'
+import Enzyme, { mount } from 'enzyme'
+import { expect } from 'chai'
+import 'regenerator-runtime/runtime'
+
+import AdminViewEditor from '../../src/Pages/Profile/admin/AdminView'
+import AdminEditorForm from '../../src/Pages/Profile/admin/EditorForm'
+import AdminProfileEditor from '../../src/Pages/Profile/admin/Profile'
+import Adapter from 'enzyme-adapter-react-16'
+import { Badge, Container, Modal, Button, FormGroup } from 'reactstrap'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+const user = {
+  accessLevel: 2,
+  firstName: 'First',
+  lastName: 'Last',
+  middleInitial: 'I',
+  joinDate: '12-32-2012htzs342',
+  membershipValidUntil: '12-32-2012htzs342',
+  email: 'email',
+  major: 'major',
+  pagesPrinted: 20
+}
+
+describe('<AdminViewEditor />', () => {
+  const wrapper = mount(<AdminViewEditor user={user} token='token' />)
+
+  it('Should render a <AdminProfileEditor /> component with one child', () => {
+    expect(wrapper.find(AdminProfileEditor)).to.have.lengthOf(1)
+  })
+
+  it('Should render a <AdminEditorForm /> component with one child', () => {
+    expect(wrapper.find(AdminEditorForm)).to.have.lengthOf(1)
+  })
+})
+
+describe('<AdminEditorForm />', () => {
+  const membership = [
+    { value: 0, name: 'Keep Same' },
+    { value: 2, name: '2 semesters' }
+  ]
+
+  const formGroups = [
+    {
+      label: 'First Name',
+      placeholder: user.firstName,
+      handleChange: () => {}
+    },
+    {
+      label: 'Last Name',
+      placeholder: user.lastName,
+      handleChange: () => {}
+    }
+  ]
+
+  const wrapper = mount(
+    <AdminEditorForm
+      formGroups={formGroups}
+      membership={membership}
+      setNumberOfSemestersToSignUpFor={() => {}}
+      setPagesPrinted={e => {}}
+      handleSubmissionToggle={() => {}}
+      handleToggle={() => {}}
+      setuserMembership={() => {}}
+      toggle
+    />
+  )
+
+  it('Should render a <Modal /> component with one child', () => {
+    expect(wrapper.find(Modal)).to.have.lengthOf(1)
+  })
+
+  it('Should render a <Button /> component with 4 children', () => {
+    expect(wrapper.find(Button)).to.have.lengthOf(4)
+  })
+
+  it('Should render a <FormGroup /> component with 8 children', () => {
+    expect(wrapper.find(FormGroup)).to.have.lengthOf(8)
+  })
+})
+
+describe('<AdminProfileEditor />', () => {
+  const wrapper = mount(
+    <AdminProfileEditor
+      user={user}
+      membershipValidUntil={user.membershipValidUntil}
+    />
+  )
+
+  it('Should render a <Badge /> component with one child', () => {
+    expect(wrapper.find(Badge)).to.have.lengthOf(1)
+  })
+
+  it('Should render a <Container /> component with one child', () => {
+    expect(wrapper.find(Container)).to.have.lengthOf(1)
+  })
+
+  it('Should render a <h3 /> component with one child', () => {
+    expect(wrapper.find('h3')).to.have.lengthOf(1)
```

Comment:
since this renders the first and last name, check that it is rendered [like this](https://github.com/SCE-Development/Core-v4/blob/af462094e8e22951b18b61f338d832419e475c38/test/frontend/UserNavbar.test.js#L47)